src/pg/test/sql/upsert.test.js

```diff
@@ -25,27 +25,31 @@ const options = {
 
 describe('byId', () => {
   test('put', async () => {
```

Comment:
Probably a good idea to add a test case for a multi-row put to verify that the optimization works